azure/scope/cluster.go

```diff
@@ -875,11 +875,17 @@ func (s *ClusterScope) APIServerHost() string {
 	return s.APIServerPublicIP().DNSName
 }
 
-// SetFailureDomain will set the spec for a for a given key.
+// SetFailureDomain to cluster's status by given id. The value of provided control plane might
```

Comment:
```suggestion
// SetFailureDomain sets a failure domain in a cluster's status by its id. The provided failure domain spec may
```

Reply:
i tried to avoid the stutter here "SetFailureDomain sets failure domain" but i didn't feel very confident about how my version read either

applied your suggestion